tests/integration/profile_test.py

```diff
@@ -273,6 +275,42 @@ def test_state_absent_can_remove_down_profiles(dummy0_with_down_profile):
         assertlib.assert_state_match(state_before_down)
 
 
+def test_create_memory_only_profile_new_interface():
+    with dummy_interface(DUMMY0_IFNAME, save_to_disk=False):
+        assert _profile_exists(
+            MEMORY_ONLY_PROFILE_DIRECTORY + "dummy0.nmconnection"
+        )
+
+    assert not _profile_exists(
+        MEMORY_ONLY_PROFILE_DIRECTORY + "dummy0.nmconnection"
+    )
+
+
+def test_create_memory_only_profile_edit_interface():
+    with dummy_interface(DUMMY0_IFNAME) as dstate:
+        assert not _profile_exists(
+            MEMORY_ONLY_PROFILE_DIRECTORY + "dummy0.nmconnection"
+        )
+        dstate[Interface.KEY][0][Interface.MTU] = 2000
+        libnmstate.apply(dstate, save_to_disk=False)
+        assert _profile_exists(
```

Comment:
This requires the knowledge of NetworkManger ifcfg plugin hence should be placed into `nm` integration test folder.
For me the old `assert_down` is enough.

Meanwhile, NM is dropping `ifcfg` plugin.

Reply:
In that case, it is important because we are checking that when modifying an active interface using memory_only, we are creating a temporal profile. I agree, profile tests should go on /nm/ integration folder, but I think it is not related to this PR. What do you think?

Reply:
Let's move in another PR. Just a trivial thing.